backend/hello.py

```
@@ -30,9 +30,10 @@ def get_lobby(code: str):
 
 @app.post("/lobby/{code}/join")
 def join_lobby(code:str, player:str):
-    lobby = lobbies.get(code)
-    if lobby is None:
-        return {"error": "Lobby not found"}
+    try:
+        lobby = lobbies[code]
+    except:
```

Never use bare except

---

This was fixed we should get this merged